Makefile

```diff
@@ -0,0 +1,181 @@
+# CMAKE generated file: DO NOT EDIT!
+# Generated by "Unix Makefiles" Generator, CMake Version 3.29
+
+# Default target executed when no arguments are given to make.
+default_target: all
+.PHONY : default_target
+
+# Allow only one "make -f Makefile2" at a time, but pass parallelism.
+.NOTPARALLEL:
+
+#=============================================================================
+# Special targets provided by cmake.
```

Comment:
Why this file being added ?

Reply:
I think this was an accidental checkin, but it suggests that you are using CMake without a separate build directory (i.e. an in-source build) which is generally not a great idea; better to create a `build/` directory and keep all build artifacts in there.